src/utils.js

```diff
@@ -267,35 +267,53 @@ exports.getPriceBucketString = function (cpm) {
   var low = '';
   var med = '';
   var high = '';
+  var auto = '';
+
   var cpmFloat = 0;
   var returnObj = {
     low: low,
     med: med,
-    high: high
+    high: high,
+    auto: auto
   };
   try {
     cpmFloat = parseFloat(cpm);
     if (cpmFloat) {
-      //round to closet .5
+      //round to closest .5
       if (cpmFloat > _lgPriceCap) {
         returnObj.low = _lgPriceCap.toFixed(2);
       } else {
         returnObj.low = (Math.floor(cpm * 2) / 2).toFixed(2);
       }
 
-      //round to closet .1
+      //round to closest .1
       if (cpmFloat > _mgPriceCap) {
         returnObj.med = _mgPriceCap.toFixed(2);
       } else {
         returnObj.med = (Math.floor(cpm * 10) / 10).toFixed(2);
       }
 
-      //round to closet .01
+      //round to closest .01
       if (cpmFloat > _hgPriceCap) {
         returnObj.high = _hgPriceCap.toFixed(2);
       } else {
         returnObj.high = (Math.floor(cpm * 100) / 100).toFixed(2);
       }
+
+      // round auto default sliding scale
+      if (cpmFloat <= 5) {
+        // round to closest .05
+        returnObj.auto = (Math.floor(cpm * 20) / 20).toFixed(2);
+      } else if (cpmFloat <= 10) {
+        // round to closest .10
+        returnObj.auto = (Math.floor(cpm * 10) / 10).toFixed(2);
+      } else if (cpmFloat <= 20) {
+        // round to closest .50
+        returnObj.auto = (Math.floor(cpm * 2) / 2).toFixed(2);
+      } else {
+        // cap at 20.00
+        returnObj.auto = 20;
```

Comment:
`20` should be a string type (with 2 places after decimal) i.e `'20.00'` since it's a string key.